Sources/Testing/EntryPoints/SwiftPMEntryPoint.swift

```diff
@@ -23,7 +23,7 @@ private import TestingInternals
 /// This function examines the command-line arguments represented by `args` and
 /// then invokes available tests in the current process.
 ///
-/// - Warning: This function is used by Swift Package Manager. Do not call it
+/// - Warning: This function is used by Swift Package Manager. Don't call it
```

Comment:
Symbols with two leading underscores aren't published in our DocC bundle, so no need to revise them.